lib/Backend/BackwardPass.cpp

```diff
@@ -2824,13 +2824,38 @@ BackwardPass::ProcessBlock(BasicBlock * block)
                 }
             }
         }
+
+        if (instr->IsLabelInstr() && instr->m_next->m_opcode == Js::OpCode::Finally)
```

Comment:
Most of the code in this block is the same as in the previous one. Please refactor into one if block.